### DISCUSSION TITLE
能否支持阿里云oss

### DISCUSSION THREAD

**Discussion by KioProject123** [AlistGo/alist#211] [Ideas]
这应该也能算个网盘吧...

**Comment by xhofe** [comment]
oss也太贵了吧😂

**Reply by hainc** [reply]
> oss也太贵了吧😂

oss确实贵，但是我觉得如果可以还是能加一个，还有啥鉴权啥的

**Comment by xhofe** [comment]
已支持S3协议存储

**Comment by songjave** [comment]
使用阿里云给的[s3兼容模式](https://help.aliyun.com/document_detail/451966.html)去访问还是报错
报错内容
`failed get objs: failed to list objs: RequestError: send request failed caused by: Get "https://rpi-home.s3.oss-cn-hangzhou.aliyuncs.com/?delimiter=%2F&marker=&prefix=": x509: certificate is valid for *.oss-cn-hangzhou.aliyuncs.com, *.img-cn-hangzhou-internal.aliyuncs.com, *.oss-cn-hzjbp-a-internal.aliyuncs.com, *.oss-cn-hangzhou-internal.aliyuncs.com, *.oss-cn-hangzhou-diku-internal.aliyuncs.com, *.cn-hangzhou.oss.aliyuncs.com, *.oss-enet-cm.aliyuncs.com, *.cn-hangzhou.oss-internal-cross.aliyun-inc.com, *.oss-cn-hangzhou-am101.aliyuncs.com, *.oss-cn-shanghai-finance-1-pub.aliyuncs.com, *.oss-cache-cn-shanghai-g.aliyuncs.com, *.oss-internal.aliyuncs.com, *.cn-shanghai.oss.aliyuncs.com, *.img-cn-shanghai.aliyuncs.com, *.oss-accelerate.aliyuncs.com, *.oss-cn-hzjbp-b-internal.aliyuncs.com, *.img-cn-hangzhou.aliyuncs.com, *.oss-cn-shanghai-finance-1-pub-internal.aliyuncs.com, *.oss-accelerate-overseas.aliyuncs.com, *.oss-cn-hangzhou-zmf.aliyuncs.com, *.oss-cn-hangzhou-hsa.aliyuncs.com, *.aliyuncs.com, *.oss.aliyuncs.com, *.oss-cn-hangzhou-cross.aliyuncs.com, *.vpc100-oss-cn-hangzhou.aliyuncs.com, *.img-cn-shanghai-internal.aliyuncs.com, *.oss-cn-hzfinance.aliyuncs.com, *.oss-cn-shanghai-finance-1-internal.aliyuncs.com, *.oss-enet.aliyuncs.com, *.cn-hangzhou.oss-internal.aliyuncs.com, *.oss-enet-cu.aliyuncs.com, *.img-cn-shanghai-cross.aliyuncs.com, *.oss-enet-ct.aliyuncs.com, *.oss-cn-shanghai.aliyuncs.com, *.oss-cn-shanghai-internal.aliyuncs.com, *.oss-cn-shanghai-cross.aliyuncs.com, *.oss-cn-shanghai-finance-1.aliyuncs.com, *.oss-cn-hangzhou-zmf-internal.aliyuncs.com, *.oss-cn-hzfinance-internal.aliyuncs.com, *.vpc100-oss-cn-shanghai.aliyuncs.com, *.cn-shanghai-finance.oss.aliyuncs.com, *.cn-hangzhou-finance.oss.aliyuncs.com, *.oss-cn-hzjbp-b-console.aliyuncs.com, *.oss-cache-cn-hangzhou-j.aliyuncs.com, *.oss-cache-cn-hangzhou-j-cross.aliyuncs.com, *.oss-cache-cn-shanghai-g-cross.aliyuncs.com, *.cn-shanghai-dualstack.oss.aliyuncs.com, *.oss-enet-cn-east.aliyuncs.com, *.cn-shanghai-vpc.oss.aliyuncs.com, *.cn-hangzhou-vpc.oss.aliyuncs.com, oss-cn-hangzhou.aliyuncs.com, not rpi-home.s3.oss-cn-hangzhou.aliyuncs.com`
